GliaWidgets/ViewModel/Call/CallViewModel.swift

```diff
@@ -98,6 +100,12 @@ class CallViewModel: EngagementViewModel, ViewModel {
         call.duration.addObserver(self) { [weak self] duration, _ in
             self?.onDurationChanged(duration)
         }
+        call.isVisitorOnHold.addObserver(self) { [weak self] isOnHold, oldValue in
```

Comment:
ViewModels seem to be like a good candidate for unit testing. Let's discuss how we can add tests here. Seems like `voice-over` branch has a lot of helpful mocks and Environments ready to be uses for testing.

Reply:
Added tests in 80bbe14e21ed66104d3535270a2bbaadb4c11327